README.md

```diff
@@ -1 +1,38 @@
-# eslint-plugin-yola
\ No newline at end of file
+# eslint-plugin-yola
+
+Extendable eslint plugin that ensures code style quality for Yola front-end codebase.  
+ES6 code style is based on [Airbnb Javascript Style Guide](https://github.com/airbnb/javascript).
+
+This plugin solves problem of shareable eslint configs that depend on eslint plugins, which described in https://github.com/eslint/eslint/issues/3458.  
```

Comment:
Documentation like this is 👍 .

Personal preference, this is best suited under an `FAQ` header. So:

```
## FAQ

### Why eslint?

We were using jshint and found it too restrictive. We wanted to extend from other style guides.

### Why a plugin?

...
```